src/components/Card.js

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 
-const Card = (props) => {
-	return (
-		<div className = 'bg-light-green dib br3 pa3 ma2 grow shadow-5 bw2' >
-			<img alt = 'robots' src = {`https://robohash.org/${props.id}tesfwedcds?200x200`}/>
```

Comment:
The image is unique to each robot, therefore, the computation of the image URL shouldn't be here